libsolidity/codegen/MultiUseYulFunctionCollector.cpp

```diff
@@ -32,16 +32,29 @@ using namespace solidity::frontend;
 string MultiUseYulFunctionCollector::requestedFunctions()
 {
 	string result;
-	for (auto const& f: m_requestedFunctions)
+	for (auto const& [name, code]: m_requestedFunctions)
 	{
-		solAssert(f.second != "<<STUB<<", "");
+		(void) name;
```

Comment:
is it possible to add a template function for that?

Reply:
https://twitter.com/ben_deane/status/624277300121632769

Reply:
```
template<typename... Args> void unused(Args const&...) {}
```
would also do it... not sure I like it though - it's really a pity that the standard doesn't account for this yet...